code/style.css

```diff
@@ -1,4 +1,119 @@
+/* Site background*/
 html {
-  background: #0025ff;
-  color: #fff;
+  background-color: #F0EEE9;
+  }
+
+/* header & it's content*/
+
+.header-container {
+  display: flex;
+  flex-wrap: wrap; 
+  }
+
+.header-logo {
+  min-width: 50%;
+  max-height: 150px;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
 }
+
+.navigation {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+  align-items: center;
+  width: 100%;
+  margin-top: 10px;
+  border-bottom: 2px groove #CF5C78;   
+  }
+
+.nav-link {
+  text-decoration: none;
+  text-transform: uppercase;
+  padding: 10px;
+  font-family: 'Concert One', cursive;
+  font-size: 16px;
+  }
+
+  /* Hover effect on navigation text*/
+  .nav-link:hover {
+    color: #CF5C78;
+  }
+
+/* Big news section & it's content*/
+.big-news-section {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  font-family: 'Concert One', cursive;
+  font-size: 38px;
+}
+
+/* Other news section & it's content*/
+  .other-news-section {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    margin-top: 10px;
+    margin-bottom: 10px;
+    margin-right: 0px;
+    margin-left: 0px;  
+    font-family: 'Concert One', cursive;
+    font-size: 22px;
+  } 
+
+/* media querie tablet - two column */
+@media (min-width: 667px) and (max-width: 1024px)  {
+  .other-news-section {
+    flex-direction: row;
+    flex-wrap: wrap;
+  } 
+}
+@media (min-width: 667px) and (max-width: 1024px) {
+  .polaroid {
+    width: 49%;
+  }
+}
```

Comment:
this is the exact same media query as above, you can but the style block for polaroid inside that as well. like this: 
```
@media (min-width: 667px) and (max-width: 1024px)  {
  .other-news-section {
    flex-direction: row;
    flex-wrap: wrap;
  } 
  .polaroid {
    width: 49%;
  }
}
```